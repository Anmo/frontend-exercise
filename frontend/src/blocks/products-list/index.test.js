import { render } from '@testing-library/react';

import ProductsList from './index';

jest.mock(
	'../../components/product',
	() =>
		function FakeProduct(props) {
			return <button {...props} />;
		}
);

test('renders the normal ProductsList block', () => {
	const { container } = render(
		<ProductsList products={['productId1', 'productId2']} />
	);

	expect(container.firstChild).toMatchSnapshot();
});
