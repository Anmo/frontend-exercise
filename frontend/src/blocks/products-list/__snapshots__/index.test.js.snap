// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the normal ProductsList block 1`] = `
<div
  class="container"
>
  <button
    productid="productId1"
  />
  <button
    productid="productId2"
  />
</div>
`;
